mypy/newsemanal/semanal.py

```diff
@@ -3810,7 +3665,7 @@ def tvar_scope_frame(self, frame: TypeVarScope) -> Iterator[None]:
 
     def lookup(self, name: str, ctx: Context,
                suppress_errors: bool = False) -> Optional[SymbolTableNode]:
-        """Look up an unqualified name in all active namespaces."""
+        """Look up an unqualified (no dots) name in all active namespaces."""
```

Comment:
Just another random idea: split the `Helpers` "section" into `Lookup methods` (and move everything related there), and `Misc helpers`.

Reply:
A good idea. I'll do this in my next refactoring PR.